.github/workflows/prepare-for-prod-ut-deploy.yml

```diff
@@ -188,11 +188,17 @@ jobs:
             git add $customer
           done
 
-          git commit -m "chore: upgrade dedicated user transformers to $TAG_NAME"
-          git push -u origin dedicated-user-transformer-$TAG_NAME
-
-          PR_URL=$(gh pr create --fill | grep -o 'https://github.com/[^/]*/[^/]*/pull/[0-9]*')
-          echo "DEDICATED_PR_URL=$PR_URL" >> $GITHUB_ENV
+          # Check if there are any changes to commit
```

Comment:
this step was an over-engineering one to handle dedicated deployments.
With the current performance issues on UT, few workspaces are on dedicated from almost a year. As long as dedicated transformers are also getting upgraded to newer version - we should be good